mlir/tools/rocmlir-gen/rocmlir-gen.cpp

```diff
@@ -223,6 +224,45 @@ static cl::opt<bool>
              cl::value_desc("To use XDLOPS V2 lowering pipeline"),
              cl::init(false));
 
+// disable feature mfma
+static cl::opt<std::string> featureInfo(
```

Comment:
I'd like to not call this "feature".

If we want this option at all, which I'm not sure we do, I want this to be something like `codepath`